src/facebook_client.py

```diff
@@ -115,10 +125,10 @@ def convert_facebook_comments_to_traced_data(user, raw_comments):
 
             traced_comments.append(TracedData(
                 {
-                    "avf_facebook_id": avf_facebook_id,  # TODO: De-identify a user's FB id here if possible instead
-                    "facebook_message_id": comment["id"],
-                    "message": comment["message"],
-                    "message_created_time": comment["created_time"],
+                    f"avf_facebook_id": f"{dataset_name}_{avf_facebook_id}",  # TODO: De-identify a user's FB id here if possible instead
```

Comment:
I think we should start testing de-identifying actual Facebook user-id here since we have admin rights to Imaqal and fallback to  `f"{dataset_name}_{avf_facebook_id}"` if we don't get it for the other pages - which seems unlikely.

Reply:
As discussed offline, this is difficult until we get the right configuration of permissions we need to see who posts are from. I won't address this now, but it remains very high priority.